docs/source/_json/registry_get_list.resp

```diff
@@ -12,26 +12,70 @@ Content-Type: application/json
   "items": [
     {
       "name": "Products.CMFPlone.i18nl10n.override_dateformat.Enabled", 
+      "schema": {
```

Comment:
@jaroel @sneridagh sorry for being too late on this. I was wondering if we want to wrap those attributes into another schema attribute. We are not using that pattern anywhere else and this is not a schema (=JSON schema). I haven't made up my mind but what do you guys think about flattening that and move description,title, and type to the top level to keep it as simple as possible?